lib/src/draggable_scrollbar.dart

```diff
@@ -123,6 +140,16 @@ class DraggableScrollbarState extends State<DraggableScrollbar> with TickerProvi
           globalPosition: Offset.zero,
           delta: Offset(0, -2),
         ));
+      else if (value.logicalKey == LogicalKeyboardKey.arrowRight)
```

Comment:
Do you think we should limit the keys to the actual scroll direction? Maybe:

    widget.scrollDirection == Axis.horizontal && value.logicalKey == LogicalKeyboardKey.arrowRight